packages/discovery-react-components/src/components/CIDocument/CIDocument.tsx

```diff
@@ -2,13 +2,13 @@ import CIDocument, {
   CIDocumentProps,
   canRenderCIDocument,
   Messages
-} from './components/CIDocument/CIDocument';
+} from '@CIDocument/components/CIDocument/CIDocument';
```

Comment:
i was hoping to just address `../../` situations. this seems more verbose than before

Reply:
Fair. I also thought about adding aliases for nested components, not just top-level ones. I think that would keep us consistently using module path aliases but reduce the verbosity. What are your thoughts on that change?